load-balancer/least-response-time/src/main/java/io/smallrye/stork/loadbalancer/leastresponsetime/LeastResponseTimeLoadBalancer.java

```diff
@@ -42,6 +46,10 @@ public ServiceInstance selectServiceInstance(Collection<ServiceInstance> service
         for (ServiceInstance instance : serviceInstances) {
             CallStatistics.CallsData callsData = callStatistics.statsForInstance(instance.getId());
             if (callsData == null || callsData.lastRecorded == CallStatistics.NO_CALL_STARTED) {
+                if (log.isDebugEnabled()) {
```

Comment:
I don't think the check is required. log.debug will do it.

Reply:
true, the ternarny operator is probably cheaper then the if